README.md

```diff
@@ -44,3 +44,24 @@ To create [Unit Tests](https://codeception.com/docs/05-UnitTests) or [Acceptance
 vendor/bin/codecept generate:suite Unit
 vendor/bin/codecept generate:suite Acceptance
 ```
+
+### Create a Test
+
+```shell
+php vendor/bin/codecept generate:cest Functional <Name>
+```
+
+### Using local code for tests
+
+Assuming you checked out module-symfony into the same directory with symfony-module-tests, you can test your branch. Add following code into composer.json and run `composer update`. Don't forget to revert composer.json and composer.lock before commit.
```

Comment:
Hmm, this seems like an interesting alternative to what we describe [here](https://github.com/Codeception/module-symfony/blob/master/CONTRIBUTING.md#1-edit-locally).

[The Contribution Guide](https://github.com/Codeception/module-symfony/blob/master/CONTRIBUTING.md) is the perfect place for this explanation, so please PR there.
So far what was suggested was to edit the module files inside the `vendor` folder of `symfony-module-tests` and then copy-paste, but this seems a bit more refined. I like.

However, if you could be a little more visual it would be great, something like this:
<hr>
*"Assuming you have the following directory structure ..:"*

```
Codeception/
├─ module-symfony/
├─ symfony-module-tests/
│  ├─ composer.json
│  ├─ composer.lock
```

<hr>